.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: go
+sudo: false
 go:
   - 1.5
   - 1.6
   - tip
 env:
   global:
-    - GO15VENDOREXPERIMENT=1
+    - GO15VENDOREXPERIMENT=1 TEST_TIMEOUT_SCALE=10
```

Comment:
Oh, cool - I didn't know that the global env vars are treated specially. Changed.